javascript/lib/api/tests/client/http/things.http.spec.ts

```diff
@@ -217,6 +228,18 @@ describe('Http Things Handle', () => {
     });
   });
 
+  it('updates the definition', () => {
```

Comment:
Please also add the corresponding 'creates the definition'-test

Reply:
I can't yet. That would need #66 to be merged.

Reply:
Done